### DISCUSSION TITLE
Evaluating alpine code in HTML retrieved from fetch request

### DISCUSSION THREAD

**Discussion by obxpete** [alpinejs/alpine#3557] [4. General]
Is this possible?  That is, have an alpine page retrieve another html page that has alpine code inside of it.  Somewhat like a iframe but using x-html instead.  

Both pages are part of our app.  We're just trying to make the 2nd page's features (all of them) work as part of multiple modules/pages of the app.  So I do have some control over the situation so if there is an advisable way of doing this please share.  

Thank you and thank you for AlpineJS!

**Comment by ekwoka** [comment]
It will just work.

The caveats being that anything using the Alpine:init event won't work since it's already inited, and any inline scripts won't evaluate like that, but all your directives and expressions will just work.
